python/ray/data/tests/test_consumption.py

```diff
@@ -1675,7 +1675,7 @@ def test_dataset_schema_after_read_stats(ray_start_cluster):
 
 
 def test_dataset_plan_as_string(ray_start_cluster):
-    ds = ray.data.read_parquet("example://iris.parquet")
+    ds = ray.data.read_parquet("example://iris.parquet", parallelism=8)
```

Comment:
This change is unrelated to the main PR, but I wanted to add this in to make the test results consistent. I found that when I run this test locally, without the explicit `parallelism` set, this splits the dataset into 20 blocks (I assume this is dependent on the compute/configs, since the test passes without issues on the CI).